### DISCUSSION TITLE
Compete by creating a Room

### DISCUSSION THREAD

**Discussion by abhinit21** [monkeytypegame/monkeytype#4047] [Ideas] [answered]
A group typing test where multiple invite-only members can join to race with time settings.
Everyone in the group receives the same words to type

**Comment by Miodec** [comment, marked as answer]
Already planned

**Reply by unrealapex** [reply]
[dev.monkeytype.com](https://dev.monkeytype.com)

**Comment by ergwastaken** [comment]
does anyone know when this is coming out?
